tests/test-pluq.C

```diff
@@ -110,14 +110,15 @@ int main(int argc, char** argv){
         tim.clear();
         tim.start();
 
-        R = FFPACK::PLUQ_basecaseCrout (F, diag, m, n, A, n, P, Q);
+        R = FFPACK::PLUQ_basecaseCrout (F, diag, (size_t)m, (size_t)n, A, (size_t)n, P, Q);
```

Comment:
why do you need a cast here?

Reply:
I will check this again if my previous modification got the compiler to complain about this.